### ISSUE TITLE
Problems with install

### ISSUE THREAD

**Issue by minzastro** [Gazorby/fish-abbreviation-tips#2]
**Describe the bug**
When I run `fisher add Gazorby/abbr-tips` I get the following error:
```
linking ~/.config/fish/conf.d/abbr_tips.fish
test: Missing argument at index 2

~/.config/fish/conf.d/abbr_tips.fish (line 65): 
   and ! test $_abbr_tips_is_spawned = 1
       ^
from sourcing file ~/.config/fish/conf.d/abbr_tips.fish
	called on line 294 of file /usr/share/fish/config.fish
from sourcing file /usr/share/fish/config.fish
	called during startup

(Type 'help test' for related documentation)
```
Plugin is not working after that too.

**System (please complete the following information):**
 - fish version 3.1.0
 - Distribution `NAME="openSUSE Tumbleweed" # VERSION="20190310"`

**Comment by Gazorby** [comment]
I messed up with 376072d1420ddecd5a63092ab7f2b5cbd2dbe4d8, it should be fixed on master with f33cf23e23de9279fb743689b0b0ae3f907d70cc

Can you confirm?

Thanks for reporting!

**Comment by Gazorby** [comment]
This should be fixed now, feel free to reopen an issue if you have problem with this again.
